backend/app/src/main/java/com/moh/phlat/backend/service/DbUtilityServiceImpl.java

```diff
@@ -202,17 +209,24 @@ public void loadProcessDataToPlr(Long controlTableId, String authenticatedUserId
 			Iterable<ProcessData> processDataList = processDataRepository
 					.getAllProcessDataByControlTableId(controlTableId);
 
-			for (ProcessData rec : processDataList) {
-				// skip record marked as DO_NOT_LOAD and send to PLR VALID records only
-				if (!"Y".equals(rec.getDoNotLoadFlag()) && RowStatusService.VALID.equals(rec.getRowstatusCode())) {
-					logger.info("loading process data with id: {} to PLR.", rec.getId());
-
-					// TO-DO call to PLR via ESB here
-					// loadPlrViaEsb(control, s);
+			if (esbBoundary.isReadyToConnect()) {
```

Comment:
if there is any error in setting up webclient, the application will not boot. Did you tried this ?

Reply:
This is mostly to catch any misconfigurations regarding the webclient, mainly with SSL bundles and certificates. That said, I haven't tested this in a while, so it might not be needed. As is, it's not negatively impacting anything, so I'm inclined to leave this as a potential item for cleanup later on